sonar-batch/src/main/java/org/sonar/batch/index/DefaultIndex.java

```diff
@@ -51,7 +57,15 @@
 import javax.annotation.Nullable;
 
 import java.io.IOException;
-import java.util.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
 
 public class DefaultIndex extends SonarIndex {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/henryju/image-hosting/master/major.png)"SonarIndex" is deprecated, extend the suggested replacement instead.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1724)